CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing to the project
+
+Everything you need to know to contribute efficiently to the project!
+
+Whatever the way you wish to contribute to the project, please respect the [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
Shall we add this code of conduct in all repos ?

Reply:
yes